mastering-plone/dexterity_2.rst

```diff
@@ -326,7 +347,8 @@ We should bind all views to it. Here is an example using the talkview.
         permission="zope2.View"
         />
 
-Note the relative Python path :py:class:`..interfaces.IPloneconfSiteLayer`. It is equivalent to the absolute path :py:class:`ploneconf.site.interfaces.IPloneconfSiteLayer`.
+Note the relative Python path :py:class:`..interfaces.IPloneconfSiteLayer`.
```

Comment:
Verify that this renders correctly. `..` looks suspicious to me. Possible double backticks needed?

Reply:
verified